spec/fixtures/coronavirus_landing_page.yml

```diff
@@ -0,0 +1,168 @@
+content:
+  title: "Coronavirus (COVID-19): what you need to do"
+  meta_description: "Find out about the government response to coronavirus (COVID-19) and what you need to do."
+  stay_at_home:
+    pretext: Stay at home
+    list:
+      - Only go outside for food, health reasons or work (where this absolutely cannot be done from home)
+      - Stay 2 metres (6ft) away from other people
+      - Wash your hands as soon as you get home
+  guidance:
+    pretext: Anyone can spread the virus.
+    link:
+      href: /government/publications/full-guidance-on-staying-at-home-and-away-from-others
+      text: Full guidance on staying at home and away from others
+  announcements_label: Announcements
+  announcements:
+    - text: You must stay at home apart from essential travel or you may be fined
+      href: /government/publications/full-guidance-on-staying-at-home-and-away-from-others
+    - text: If you live in the UK and are currently abroad you are strongly advised to return now
+      href: /guidance/travel-advice-novel-coronavirus
+    - text: All non-essential shops and community spaces are closed
+      href: /government/publications/further-businesses-and-premises-to-close
+  nhs_banner:
+    heading: "Do not leave home if you or someone you live with has either:"
+    list:
+      - a high temperature
+      - a new, continuous cough
+    call_to_action:
+      title: Check the NHS website if you have symptoms
+      href: https://www.nhs.uk/conditions/coronavirus-covid-19/
+  explainer_title:
```

Comment:
bye bye explainer_title and explainer_text ✂️

Reply:
Yep I left them out of the keys for validation!